Week-9/todoapplication/src/components/Todo.js

```diff
@@ -0,0 +1,44 @@
+import React, { useContext } from "react";
+import { TodoContext } from "./";
+import PropTypes from "prop-types";
+// import { _noop } from "lodash/noop";
+
+import "../AppWideCSS.css";
+
+function Todo({ todo: { id }, todo: { isDone }, todo: { title }, todo }) {
```

Comment:
todo: {id, isDone, title}.

Reply:
This stuff is new to me...
Yes, addressed this.